backend/app/views/spree/admin/promotion_rules/create.js.erb

```diff
@@ -4,6 +4,12 @@ $('#rules .no-objects-found').hide();
 $('.product_picker').productAutocomplete();
 $('.user_picker').userAutocomplete();
 
+$('select.select2').select2({
```

Comment:
I'm not a frontend expert by any means, but is it strange that we have to have this bit of code here?  We select all `select.select2`s on the whole page and apply some settings to them?  Should the scope be limited at all?  Maybe I'm just not familiar with how we're doing this sort of thing normally.